compiler/esy.json

```diff
@@ -44,25 +44,27 @@
     "@opam/ppx_deriving_cmdliner": ">= 0.6.0",
     "@opam/ppx_deriving_yojson": ">= 3.5.2",
     "@opam/ppx_sexp_conv": ">= 0.14.0",
-    "@opam/reason": ">= 3.6.2",
+    "@opam/reason": ">= 3.8.0",
     "@opam/sexplib": ">= 0.14.0",
     "@opam/uri": ">= 4.2.0 < 5.0.0",
     "@opam/utf8": "0.1.0",
     "@opam/yojson": ">= 1.7.0",
-    "ocaml": "4.12.0"
+    "ocaml": "4.14.0"
   },
   "devDependencies": {
-    "@opam/js_of_ocaml-compiler": ">= 3.11.0 < 4.0.0",
-    "@opam/ocaml-lsp-server": "1.8.3",
+    "@opam/js_of_ocaml-compiler": ">= 4.0.0 < 5.0.0",
+    "@opam/ocaml-lsp-server": "1.10.6",
     "@reason-native/rely": "^4.0.0"
   },
   "resolutions": {
-    "ocaml": "4.12.0",
-    "@opam/fp": "reasonml/reason-native:fp.opam#0ed854f986256e52e59aeecfa90e9af60f105b15",
-    "@opam/fs": "reasonml/reason-native:fs.opam#0ed854f986256e52e59aeecfa90e9af60f105b15",
-    "@opam/utf8": "reasonml/reason-native:utf8.opam#0ed854f986256e52e59aeecfa90e9af60f105b15",
-    "@reason-native/cli": "reasonml/reason-native:cli.json#0ed854f986256e52e59aeecfa90e9af60f105b15",
-    "@reason-native/rely": "reasonml/reason-native:rely.json#0ed854f986256e52e59aeecfa90e9af60f105b15"
+    "ocaml": "4.14.0",
+    "@opam/fp": "ospencer/reason-native:fp.opam#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@opam/fs": "ospencer/reason-native:fs.opam#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@opam/utf8": "ospencer/reason-native:utf8.opam#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@reason-native/cli": "ospencer/reason-native:cli.json#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@reason-native/rely": "ospencer/reason-native:rely.json#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@reason-native/pastel": "ospencer/reason-native:pastel.json#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6",
+    "@reason-native/file-context-printer": "ospencer/reason-native:file-context-printer.json#8d2b9c3cbf1d0a4074f34116e7e92ccb00b040f6"
```

Comment:
@phated Reason native hasn't upgraded their Reason version yet—I plan on submitting a PR over there, but should we have a @grain-lang fork instead of my personal one?

Reply:
I think it would look better to have the forked deps under grain-lang 👍

Reply:
All set!